ReactWindows/ReactNative.Shared/Bridge/JavaScriptBundleLoader.cs

```diff
@@ -100,7 +100,7 @@ public override Task InitializeAsync()
                 var assembly = Assembly.GetAssembly(typeof(JavaScriptBundleLoader));
                 var assemblyName = assembly.GetName();
                 var pathToAssembly = Path.GetDirectoryName(assemblyName.CodeBase);
-                var pathToAssemblyResource = Path.Combine(pathToAssembly, SourceUrl.Replace("ms-appx:///", String.Empty));
+                var pathToAssemblyResource = Path.Combine(pathToAssembly, SourceUrl);
                 var u = new Uri(pathToAssemblyResource);
```

Comment:
Why do you still need to do a path.Combine here?  Isn't the SourceUrl now based on the initial bundle URL?